configure.ac

```diff
@@ -564,6 +575,26 @@ if test "$gssapi" != "no"; then
      AC_DEFINE(GSS_USE_MUTEXES, [], [should we mutex-wrap calls into the GSS library?])
   fi
   AC_MSG_RESULT($use_gss_mutexes)
+
+  AC_MSG_CHECKING(to use GSS delgation with ccpath)
+  AC_ARG_ENABLE(gss_cc, [  --enable-gss-cc         allow delegation of cached crdentials],
+                use_gss_cc=$enableval,
+                use_gss_cc=$cc_default)
+
+  if test $use_gss_cc = "yes"; then
```

Comment:
GSS_USE_CC is very vague, I would use a more descriptive define like GSS_STORE_DELEGATED_CREDS